feature/account/server/settings/src/main/kotlin/app/k9mail/feature/account/server/settings/domain/usecase/ValidatePassword.kt

```diff
@@ -10,12 +10,17 @@ class ValidatePassword : UseCase.ValidatePassword {
     override fun execute(password: String): ValidationResult {
         return when {
             password.isBlank() -> ValidationResult.Failure(ValidatePasswordError.EmptyPassword)
-
+            password.contains(LINE_BREAK) -> ValidationResult.Failure(ValidatePasswordError.LinebreakInPassword)
             else -> ValidationResult.Success
         }
     }
 
     sealed interface ValidatePasswordError : ValidationError {
         data object EmptyPassword : ValidatePasswordError
+        data object LinebreakInPassword : ValidatePasswordError
+    }
+
+    private companion object {
+        const val LINE_BREAK = "\n"
```

Comment:
Is is possible to have a password that contains \r followed by a space for example?
I am unsure anyone could even copy paste this 🤔

Reply:
I tried that but it seems to be converted to "\n".